.circleci/config.yml

```diff
@@ -11,111 +19,136 @@ commands:
         failure_message: ':red_circle: M1: A $CIRCLE_JOB job has failed!'
         only_for_branches: master
         webhook: $SLACK_GREENKEEPER_WEBHOOK
-
 jobs:
   unit-php55-mage18:
     docker:
-      - image: bylexus/apache-php55
-      - image: circleci/mysql:5.7
+    - image: bylexus/apache-php55
```

Comment:
please add a tech debt task to use a circle convenience image instead of an image we dont know who maintains

Reply:
Added